modules/serverless-kafka-tls-sasl.adoc

```diff
@@ -0,0 +1,87 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+= Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the `KafkaChannel` to work against a protected Kafka cluster using TLS or SASL.
+
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A CA certificate as a `.pem` file.
+* A client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
+
+. Reference your secret and the namespace of the secret in the `config-kafka` ConfigMap:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-kafka
+  namespace: knative-eventing
+data:
+  bootstrapServers: <bootstrap-servers>
+  authSecretName: <kafka-auth-secret>
```

Comment:
```suggestion
  bootstrapServers: <list_of_bootstrap_servers>
  authSecretName: <kafka_auth_secret>
```
If it's multiple bootstrap servers I'd mention it as a list, also the OCP doc guidelines use underscores now afaik for variables